tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38,py39,py310,py311,py312,mypy,docs,twinecheck,min
+envlist = min,min-extra,py38,py39,py310,py311,py312,extra,mypy,docs,twinecheck
```

Comment:
I think we should include new environments to github actions, so that they're executed on pull requests